src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -342,19 +342,40 @@ void sendProductEvent(String event, String category, Properties properties) {
       return;
     }
 
-    ItemHitBuilder itemHitBuilder = new ItemHitBuilder();
-    itemHitBuilder.setTransactionId(properties.orderId())
-        .setCurrencyCode(properties.currency())
+    com.google.android.gms.analytics.ecommerce.Product product =
+            new com.google.android.gms.analytics.ecommerce.Product()
+        .setId(properties.productId())
         .setName(properties.name())
-        .setSku(properties.sku())
         .setCategory(isNullOrEmpty(category) ? DEFAULT_CATEGORY : category)
         .setPrice(properties.price())
-        .setQuantity(properties.getLong(QUANTITY_KEY, 0))
-        .build();
-    attachCustomDimensionsAndMetrics(itemHitBuilder, properties);
-    Map<String, String> itemHit = itemHitBuilder.build();
-    tracker.send(itemHit);
-    logger.verbose("tracker.send(%s);", itemHit);
+        .setQuantity(properties.getInt(QUANTITY_KEY, 0));
```

Comment:
Discussed in person - properties.quantity is meant to be passed in as an int/number according to our spec, so this should be o.k.